soc/arm/atmel_sam0/samd20/CMakeLists.txt

```diff
@@ -3,4 +3,5 @@
 # Copyright (c) 2018 Sean Nyekjaer
 # SPDX-License-Identifier: Apache-2.0
 
+zephyr_include_directories(${ZEPHYR_BASE}/drivers)
 zephyr_sources(soc.c)
```

Comment:
This file got moved to `common/`

Reply:
ACK, rebased and moved to `soc/arm/atmel_sam0/CMakeLists.txt` (since it's only added a a sub-directory for SAM0 series anyway).